core/consensus/src/synchronization.rs

```diff
@@ -95,9 +99,15 @@ impl<Adapter: SynchronizationAdapter> Synchronization for OverlordSynchronizatio
         )?;
 
         log::info!(
-            "[synchronization]: end, remote block height {:?} current block height {:?}",
+            "[synchronization]: sync end, remote block height {:?} current block height {:?} current exec height {:?} current proof height {:?}",
             remote_height,
-            current_height,
+            self.status.to_inner().current_height,
+            self.status.to_inner().exec_height,
+            self.status.to_inner().current_proof.height,
+        );
+        log::info!(
+            "[synchronization]: sync end, current consented status :{:?}",
+            self.status.to_inner()
```

Comment:
Call `to_inner()` function, which will get `ReadLockGuard`, too many times.

Reply:
yep, reduce the use